### DISCUSSION TITLE
Insane amount of spam ads in Community Discussions.

### DISCUSSION THREAD

**Discussion by davevad93** [community/community#126000] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi everyone I hope y'all are doing well, 

I want to address ***a very big issue*** I'm seeing here in the ***Community Discussions*** and that's the ***amount of spam*** which to me is insane. 

For example in the last few hours I've counted almost ***30 spam discussions*** and the majority are ***airline companies' ads***.
I've noticed that the accounts posting those discussions are ***less than 1 day old***, so I'm wondering if you don't have some algorithm or something to prevent things like that.

I think a starting solution might be to disallow opening discussions to accounts that are ***less than 1 week old***, because I've noticed that those posts are made by new "bot" accounts that have joined GitHub in the ***last few hours***.

I know that this idea has many flaws because it would't be very fair to new users who want to interact fairly with the Community but the truth is that every day there are ***too many spam discussions*** and that in my opinion it truly ***deteriorates*** the ***quality*** and the ***environment*** of the Community.

I hope that some kind of action can be taken.

Thanks in advance, have a nice day!

**Comment by m4xy07** [comment]
Could automoderate posts with a few keywords that I spot in most of the spam posts on here, Mobile Numbers, also add a check so that recently created accounts posts are limited/flagged for moderation and then approved. 

The current amount of spam is insane

**Comment by akathedeveloper** [comment]
@davevad93 
I've noticed a surge in spam discussions, particularly ads from new accounts. It's disruptive and impacts community quality. While restricting new accounts from posting for a week may help, it could also hinder genuine users. Combining automated filtering with manual moderation may be more effective. Hoping GitHub addresses this soon.

**Comment by ossanna16** [comment, marked as answer]
Hi there @davevad93 and thank you for the feedback! 

We are aware of the amount of spam in the Community and are doing our best to keep this space clean and tidy for our community members within our current means. We are also constantly discussing internally on how we can improve our spam fighting efforts - unfortunately it is not very straightforward and there are a lot of pieces involved in figuring out this problem. Thank you for your patience with us whilst we are trying to address this issue.

**Reply by davevad93** [reply]
Hi @ossanna16 ,

Thank you for the reply, I understand that it's difficult to fix it straightforwardly and I'm sure y'all are working hard to find a suitable solution, so for now, I will keep reporting all the spam I spot around here.

**Reply by ossanna16** [reply]
Thank you so much! We really appreciate the help 🚀

**Reply by davevad93** [reply]
Hi again @ossanna16 , in the last month and a half I've noticed that spam has decreased drastically, I no longer see discussions of airline advertising or similar stuff. I can tell that you have taken the matter seriously. I want to thank you and the whole team for fixing this because it was really unbearable to see an average of 15-20 spam discussions every day.

Thank you very much!

**Reply by ossanna16** [reply]
Thank you for the kind feedback @davevad93!

**Comment by Xelliese** [comment]
Hey everyone,

I've noticed a huge increase in spam posts lately, especially from new accounts less than a day old. Maybe GitHub could consider not allowing new accounts to start discussions for the first week to combat this. It's just a thought, but something definitely needs to be done to maintain the quality of our community.

**Reply by ossanna16** [reply]
Thanks @Xelliese! That is a solution we are considering and we are working hard towards fighting spam on this platform as we are aware that it impacts user experience.
